docs/tutorial.md

```diff
@@ -110,11 +153,12 @@ Parent commit: kntqzsqt 5d39e19d Say goodbye
 ```
 
 If we later realize that we want to make further changes, we can make them
-in the working copy and then run `jj squash`. That command squashes the changes
-from a given commit into its parent commit. Like most commands, it acts on the
-working-copy commit by default. When run on the working-copy commit, it behaves
-very similar to `git commit --amend`, and `jj amend` is in fact an alias for
-`jj squash`.
+in the working copy and then run `jj squash`.
+That command squashes (moves) the changes from a given commit into its
+parent commit.
+Like most commands, it acts on the working-copy commit by default.
+When run on the working-copy commit, it behaves very similar to
+`git commit --amend`, and `jj amend` is in fact an alias for `jj squash`.
```

Comment:
nit: I liked the old formatting better. I don't think it's a common convention to start each sentence on a new line in Markdown files, is it? This applies throughout the file. I would probably just rely on the automatic formatting. Which tool did you use for that, btw?

Reply:
It’s quite common, since it produces better diffs and fewer conflicts, and makes sentence‐level operations easier to do in editors. We do it in the Nix ecosystem. See e.g. https://rhodesmill.org/brandon/2012/one-sentence-per-line/, https://nick.groenen.me/notes/one-sentence-per-line/.

Reply:
Interesting! I'd be fine with switching to that style consistently. Maybe formatters can be configured to do it too?

Reply:
Yeah, sorry. I used a hybrid style. I'm also used to the "one sentence per line"-style but saw that jj is not using that convention "yet", so I only used it where I believed it'd make sense as I had weirdly split sentences otherwise.

I used the built-in editor from IntelliJ IDEA to format the file and then did the rest manually.

I'm happy to follow whatever you prefer. If you like I could reformat the whole file in this style (and could look at the rest of the docs in a follow-up PR).

Reply:
Let's wait a bit to hear what others think about changing the style so you don't waste time doing it and then people don't want it.

Reply:
Repeating what I said on Discord:

I wouldn't change the markdown style, or be too strict about it, until we have a tool that automatically formats markdown. Then, we'll be a bit limited by what our tool of choice supports. Some of them support one-sentence-per-line, I think, but not all.

Reply:
For another perspective, I, personally, use the ReWrap plugin or the `fmt` shell command to reformat paragraphs. So, making it one sentence per line without proper tooling would make my life harder. (Though I could live with it if needed)

With proper tooling, I'd be interested in trying it out and see how it improves diffs.

Reply:
Dprint [does not seem to support one-sentence-per-line](https://dprint.dev/plugins/markdown/config/), though there is an option to prevent it from wrapping text.

Prettier seems to support [one paragraph per line](https://prettier.io/docs/en/options#prose-wrap), but not one-sentence-per-line.

Reply:
Okay, so for now I have just reformated everything back to the original. I don't have a strong opinion.

Done in https://github.com/martinvonz/jj/commit/b2692a325f251969f212153df4d02b26b772efb0